File: 1503-last-moment-before-all-ants-fall-out-of-a-plank/1503-last-moment-before-all-ants-fall-out-of-a-plank.java
class Solution {
    public int getLastMoment(int n, int[] left, int[] right) {
        int lastMoment = 0;
        for(int i=0; i<left.length; i++){
            lastMoment = Math.max(lastMoment, left[i]);
        }
        for(int i=0; i<right.length; i++){
            lastMoment = Math.max(lastMoment, n-right[i]);
        }
        return lastMoment;
    }

}

File: 1503-last-moment-before-all-ants-fall-out-of-a-plank/1503-last-moment-before-all-ants-fall-out-of-a-plank.cs
public class Solution {
    public int GetLastMoment(int n, int[] left, int[] right) {
        int lastMoment = 0;
        for(int i=0; i<left.Length; i++){
            lastMoment = Math.Max(lastMoment, left[i]);
        }
        for(int i=0; i<right.Length; i++){
            lastMoment = Math.Max(lastMoment, n-right[i]);
        }
        return lastMoment;
    }
}